### PR TITLE
Add workshop DOIs to doi_license.yml

### PR DESCRIPTION
**Pull request by jlpalomino**
@lwasser This pull request adds the Zenodo DOIs for the five earth analytics workshops to the doi_license.yml. I built locally but the workshop main pages did not appear updated with the DOI information. Let me know if I need to do something else to update these pages.

### CODE REVIEW

**Comment by lwasser** [comment]
phew... there are a lot of bad links here @jlpalomino that seem to be related to older pages that you did not edit. i'm wondering if we should put an intern on this to clean things up. 
i'm going to think about this a bit more ... and in the meantime thank you for submitting this!

File: _data/doi_license.yml [hunk 1]
```diff
@@ -4,3 +4,13 @@
   doi: https://zenodo.org/record/1326873#.XA_xnCdRcWo
```
Comment (lwasser) [inline _data/doi_license.yml]: @jlpalomino just curious. why is the format of this DOI url different from the ones below?

Comment (jlpalomino) [reply _data/doi_license.yml]: @lwasser I am not sure; I believe you added those DOI urls in PR #715.  Would like me to change this one and the one for earth-analytics-bootcamp to the latest doi badge?

Comment (lwasser) [reply _data/doi_license.yml]: thanks @jlpalomino  yes please. i think at the time, i didn't know the difference between the DOI links in zenodo! Please update them in this PR.
Thank you!!

Comment (jlpalomino) [reply _data/doi_license.yml]: @lwasser Done!

**Pull request merged**